src/plugins/azgenerator/CodeModelAz.ts

```diff
@@ -105,4 +105,6 @@ export interface CodeModelAz
     Example_Title: string;
     Example_Params: any;
     GetExamples(): CommandExample[];
+    FormatPreparers(imports: string[], decoraters: string[]);
```

Comment:
i think FormatPreparers should not be a part of codemoded, as it does actual code formatting.
CodeModel should just provide essential information.